### PR TITLE
Added changelog for v2.1.0

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
+## 2.1.0 (October 20, 2015)
+### general
+ - Added ability to install and upgrade Logstash plugins without requiring internet connectivity (#2376). 
+ - Support alternate or private Ruby gems server to install and update plugins (#3576).
+ - Fixed a memory leak which could be triggered when events having a date were serialized to string (#4222).
+ - Added JDBC input to default package.
+ - Logstash can read multiple config files from a directory. If you combine --config_test CLI flag with
```
Comment (jsvd) [inline CHANGELOG.md]: Maybe simpler this way?
`Adding --debug to --configtest now shows the configuration in blocks annotated by file. Very useful when using multiple config files in a directory.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
+## 2.1.0 (October 20, 2015)
+### general
+ - Added ability to install and upgrade Logstash plugins without requiring internet connectivity (#2376). 
+ - Support alternate or private Ruby gems server to install and update plugins (#3576).
+ - Fixed a memory leak which could be triggered when events having a date were serialized to string (#4222).
+ - Added JDBC input to default package.
+ - Logstash can read multiple config files from a directory. If you combine --config_test CLI flag with 
+   `--debug`, Logstash will log the combined config file, annotating the individual config blocks
+   with the source file it came from (#3243)
+ - Reset default worker threads to 1 when using non thread-safe filter like multiline (#4130).
```
Comment (jsvd) [inline CHANGELOG.md]: s/filter/filters

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+   Twitter
+     - Added an option to fetch data from the sample Twitter streaming endpoint (#21).
+     - Added hashtags, symbols and user_mentions as data for the non extended tweet event (#22).
+     - Added an option to filter per location and language (#20).
+     - Added an option to stream data from a list of users (#11).
+   Beats
+     - Properly handle multiline events from multiple sources, originating from Filebeat (#10).
+
+   File
+     - Properly handle multiline events from multiple sources.
```
Comment (jsvd) [inline CHANGELOG.md]: (#44)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+
+   Eventlog
+     - Change the underlying library to capture Event Logs from Windows more reliably.
```
Comment (jsvd) [inline CHANGELOG.md]: (#11)

File: CHANGELOG.md (hunk 1 above)
Comment (jordansissel) [inline CHANGELOG.md]: (also noting that it's `--configtest` not `--config_test`)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+ - Added JDBC input to default package.
+ - Logstash can read multiple config files from a directory. If you combine --config_test CLI flag with 
+   `--debug`, Logstash will log the combined config file, annotating the individual config blocks
+   with the source file it came from (#3243)
+ - Reset default worker threads to 1 when using non thread-safe filter like multiline (#4130).
+ - Fixed file permissions for the logrotate configuration file.
+ - Changed the default heap size from 500MB to 1GB (#3861)
+ - Fixed config check option when starting Logstash through init scripts (#3645)
+
+ ### input
```
Comment (jsvd) [inline CHANGELOG.md]: this section is indented one level, which makes input/output sections misaligned with general

**Comment by jsvd** [comment]
LGTM

**Comment by guyboertje** [comment]
LGTM

**Comment by jsvd** [comment]
This should be merged against 2.1 and 2.x branches

**Comment by jsvd** [comment]
Raised some concerns about CHANGELOG.md inconsitency in https://github.com/elastic/logstash/issues/4251

**Comment by jsvd** [comment]
merged in 2.x f874bb4
merged in 2.1 5ddbb32

**Pull request closed without merging**